client/src/lit/compat/bcd-table.js

```diff
@@ -0,0 +1,801 @@
+        <span
+          class="bc-version-label"
+          title=${browserReleaseDate && !timeline
+            ? `${browser.name} ${added} – Released ${browserReleaseDate}`
+            : ""}
+        >
+          ${timeline && false ? browser.name : null}
+          ${!timeline || browserReleaseDate ? label : null}
+          ${browserReleaseDate && timeline
+            ? ` (Released ${browserReleaseDate})`
+            : ""}
+        </span>
+      </div>
+      ${support && this.renderCellIcons(support)}
+    </div>`;
+  }
+
+  renderTableLegend() {
+    const { browserInfo } = this;
+
+    if (!browserInfo) {
+      throw new Error("Missing browser info");
+    }
+
+    return html`<section class="bc-legend">
+      <h3 class="visually-hidden" id="Legend">Legend</h3>
+      <p class="bc-legend-tip">
+        Tip: you can click/tap on a cell for more information.
+      </p>
+      <dl class="bc-legend-items-container">
+        ${getActiveLegendItems(this.data, this.name, browserInfo).map(
+          ([key, label]) =>
+            ["yes", "partial", "no", "unknown", "preview"].includes(key)
+              ? html`<div class="bc-legend-item">
+                  <dt class="bc-legend-item-dt">
+                    <span class=${`bc-supports-${key} bc-supports`}>
+                      <abbr
+                        class=${`bc-level bc-level-${key} icon icon-${key}`}
+                        title=${label}
+                      >
+                        <span class="visually-hidden">${label}</span>
+                      </abbr>
+                    </span>
+                  </dt>
+                  <dd class="bc-legend-item-dd">${label}</dd>
+                </div>`
+              : html`<div class="bc-legend-item">
+                  <dt class="bc-legend-item-dt">
+                    <abbr
+                      class="legend-icons icon icon-${key}"
+                      title=${label}
+                    ></abbr>
+                  </dt>
+                  <dd class="bc-legend-item-dd">${label}</dd>
+                </div>`
+        )}
+      </dl>
+    </section>`;
+  }
+
+  render() {
+    return html`
+      ${this.renderIssueLink()} ${this.renderTable()}
+      ${this.renderTableLegend()}
+    `;
+  }
+}
+
+customElements.define("bcd-table", BcdTable);
+
+/**
+ * Return a list of platforms and browsers that are relevant for this category &
+ * data.
+ *
+ * If the category is "webextensions", only those are shown. In all other cases
+ * at least the entirety of the "desktop" and "mobile" platforms are shown. If
+ * the category is JavaScript, the entirety of the "server" category is also
+ * shown. In all other categories, if compat data has info about Deno / Node.js
+ * those are also shown. Deno is always shown if Node.js is shown.
+ *
+ * @param {string} category
+ * @param {import("@mdn/browser-compat-data/types").Identifier} data
+ * @param {import("@mdn/browser-compat-data/types").Browsers} browserInfo
```

Comment:
I'd move these to the
```
/**
 * @import ...
 * ...
 * /
```
block at the top

Reply:
Fixed in 96438547ca08326dfec7fd29c14241b39423a839.